Source/Scene/Cesium3DTileset.js

```diff
@@ -113,6 +133,11 @@ define([
      * @param {Boolean} [options.debugShowViewerRequestVolume=false] For debugging only. When true, renders the viewer request volume for each tile.
      * @param {Boolean} [options.debugShowGeometricError=false] For debugging only. When true, draws labels to indicate the geometric error of each tile
      * @param {ShadowMode} [options.shadows=ShadowMode.ENABLED] Determines whether the tileset casts or receives shadows from each light source.
+     * @param {Boolean} [options.skipLODs=true] Determines if level-of-detail skipping optimization should be used.
```

Comment:
Probably make a new class with all readonly getters to keep it simple.  It will require some rework.  I think it is worth it, but up to you.